content/docs/sidebar.json

```diff
@@ -98,7 +98,12 @@
           "katacoda": "https://katacoda.com/dvc/courses/examples/dvcignore"
         }
       },
-      "updating-tracked-files",
+      {
+        "label": "How To",
+        "slug": "how-to",
+        "source": false,
+        "children": ["undo-adding-data", "update-tracked-files"]
```

Comment:
So! It seems this URL transition was not smooth at all. Just leaving this here for future reference and it's not a major page that used to have thousands of visits but we went from over 100 to around 20 total after this 😢 (for now).

A redirect would probably have helped after all (cc @shcheklein) because some the traffic for this page came from old GitHub comments that had a direct link, which after this change started producing 404s (I fixed those comments now).

![image](https://user-images.githubusercontent.com/1477535/95254409-7a8e8d80-07e5-11eb-8444-ced470078049.png)

> Cc thanks for the tip on GA usage for this @jeremydesroches